pipelines/main/platforms/upload_linux.arches

```diff
@@ -1,9 +1,9 @@
-# OS       TRIPLET                  TIMEOUT
```

Comment:
This broke the upload jobs?

Reply:
Yes.

Reply:
https://github.com/JuliaCI/julia-buildkite/pull/170